### PR TITLE
Add blog post for release 1.3.1

### PR DESCRIPTION
**Pull request by PARitter**
Signed-off-by: Phillip Ritter <pritter@fb.com>

Blog post for minor release 1.3.1

### CODE REVIEW

**Comment by PARitter** [comment]
PR submitted prematurely.  Further comments to release notes pending.  Will resubmit when clarified.

**Comment by PARitter** [comment]
Updated blog post.  Ready for review.

File: src/pages/blog/magma-release-1.3.1.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+templateKey: blog-post
+title: Magma Release 1.3.1
+author: Phil Ritter
+date: 2020-11-10 T09:00:00.000Z
```
Comment (jbryce) [inline src/pages/blog/magma-release-1.3.1.md:5]: The space before T will cause the template rendering to fail unable to parse the timestamp.

File: src/pages/blog/magma-release-1.3.1.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+category:
+  - label: Release Notice
+    id: release-notice
+---
+# Magma v1.3.1
+
+We are please to announce the v1.3.1 hotfix release for Magma.  This release contains support for unmanaged eNobeB as well as fixes for a few known issues. See the [release page](https://github.com/magma/magma/releases/tag/v1.3.1) for more information and release notes.
```
Comment (jbryce) [inline src/pages/blog/magma-release-1.3.1.md:12]: 2 typos
- "We are please" -> "We are pleased"
- "eNobeB" -> "eNodeB"

**Review by jbryce — CHANGES_REQUESTED** [review]

**Comment by jbryce** [comment]
Also note, you will probably need to do an upstream fetch to get the latest changes from https://github.com/magma/magma-website/pull/34 that fix the other dependency failure.

File: src/pages/blog/magma-release-1.3.1.md (hunk 1 above)
Comment (PARitter) [reply src/pages/blog/magma-release-1.3.1.md]: Fixed timestamp

**Comment by PARitter** [comment]
Corrected the timestamp and fixed the timestamp.  Should be ready now.

**Review by jbryce — APPROVED** [review]

**Pull request merged**